internal/core/testdata/test-template-printer-template-list-with-options-unknown-column.golden

```diff
@@ -0,0 +1,19 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Templating error
+
+Hint:
+Acceptable values are:
+  - ID
+  - Name
+🟩🟩🟩 JSON STDOUT 🟩🟩🟩
+[
```

Comment:
It's confusing to have JSON in goldens for such test

Reply:
I don't think we can avoid it. @jerome-quere